service/worker/deployment/deployment_workflow.go

```diff
@@ -300,34 +302,15 @@ func (d *DeploymentWorkflowRunner) handleSyncState(ctx workflow.Context, args *d
 		}
 
 		// sync to task queues
-		syncReq := &deploymentspb.SyncUserDataRequest{
-			Deployment: d.State.Deployment,
-		}
-		for tqName, byType := range d.State.TaskQueueFamilies {
-			for tqType, data := range byType.TaskQueues {
-				syncReq.Sync = append(syncReq.Sync, &deploymentspb.SyncUserDataRequest_SyncUserData{
-					Name: tqName,
-					Type: enumspb.TaskQueueType(tqType),
-					Data: d.dataWithTime(data),
-				})
-			}
+		if workflow.GetVersion(ctx, "syncToTaskQueues", workflow.DefaultVersion, 0) == workflow.DefaultVersion {
+			err = d.syncToTaskQueues1(ctx)
```

Comment:
Nit: should we call these `syncToTaskQueues0` and `syncToTaskQueues1` to match the version?